addons/project/res_config_view.xml

```diff
@@ -24,15 +22,7 @@
                               </div>
                          </div>
                     </group>
-                    <group string="Timesheets">
-                        <label for="id" string="Extra features"/>
-                        <div>
-                            <field name="module_project_timesheet_synchro" widget="upgrade_boolean"/>
-                            <label for="module_project_timesheet_synchro"/>
-                            <a href="https://www.odoo.com/page/timesheet" target="_blank"> More Info </a>
-                        </div>
-                        <field name="module_project_issue_sheet" widget="radio"/>
-                    </group>
+                    <group name="timesheets"/>
```

Comment:
@tivisse ,
Yes, because , In timesheets group . we have to different field and shifted in two different module which are not depended on each-other. 

so when hr_timesheet_sheet install we need to make "module_project_timesheet_synchro" visible in project configuration .
and when project_issue installed , we need to make "module_project_issue_sheet" inside timesheets group with string.